extensions/common/vault/vault-hashicorp/src/main/java/org/eclipse/edc/vault/hashicorp/HashicorpVaultExtension.java

```diff
@@ -71,6 +71,9 @@ public class HashicorpVaultExtension implements ServiceExtension {
     @Setting(value = "The URL path of the vault's /secret endpoint", defaultValue = VAULT_API_SECRET_PATH_DEFAULT)
     public static final String VAULT_API_SECRET_PATH = "edc.vault.hashicorp.api.secret.path";
 
+    @Setting(value = "The path of the folder that the secret is stored in", required = false, defaultValue = "")
+    public static final String VAULT_FOLDER_PATH = "edc.vault.hashicorp.folder";
```

Comment:
Should we maybe name it subfolder?

Reply:
The question is whether the path is relative or absolute, not if it is a subfolder

Reply:
I guess it depends on how you define relative/absolute in this case, right? 😄

For example the "full" URL would be: `https://the-vault.com/v1/secrets/data/some/deep/deep/deep/folder/my-secret`
This url splits up in the following parts:
- `https://the-vault.com/v1/secrets` (already covered by `VAULT_URL` and `VAULT_API_SECRET_PATH`)
- `data` (or `metadata`)
- `some/deep/deep/deep/folder` (will be covered by `VAULT_FOLDER_PATH`
- `my-secret`

Reply:
An absolute path always starts with a scheme, so I don't think there is any ambiguity here. From the example, the path is always relative so it should explicitly state that, "The path of the folder relative to VAULT_FOLDER_PATH..."

Reply:
@saschaisele can you please align the changes

Reply:
changed the description to reflect the relative nature of the path